### PR TITLE
Avoid usage of ClassUtils

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Fixed
- Avoid dependency of `doctrine/common`
```

### CODE REVIEW

File: src/Model/ModelManager.php [hunk 1]
```diff
@@ -236,9 +235,8 @@ public function executeQuery(object $query)
 
     public function getIdentifierValues(object $model): array
     {
-        $class = ClassUtils::getClass($model);
-        $metadata = $this->getMetadata($class);
-        $platform = $this->getEntityManager($class)->getConnection()->getDatabasePlatform();
+        $metadata = $this->getMetadata($model::class);
+        $platform = $this->getEntityManager($class::class)->getConnection()->getDatabasePlatform();
```
Comment (jordisala1991) [inline src/Model/ModelManager.php]:
```suggestion
        $platform = $this->getEntityManager($model::class)->getConnection()->getDatabasePlatform();
```

File: tests/Model/ModelManagerTest.php [hunk 2]
```diff
@@ -430,18 +438,25 @@ public function testGetEntityManagerException(): void
...
 
-        $entityManger->expects(static::once())
+        $entityManager->expects(static::once())
             ->method('persist');
 
-        $entityManger->expects(static::once())
+        $entityManager->expects(static::once())
             ->method('flush')
             ->willThrowException($exception);
+
```
Comment (jordisala1991) [inline tests/Model/ModelManagerTest.php:456]:
```suggestion

```

**Comment by jordisala1991** [comment]
Wdyt @VincentLanglet ? This might avoid usage of ClassUtils here (we dont even have the explicit dep on composer.json, thats why I havent remove it)

**Review by VincentLanglet — APPROVED** [review]
Do we have a functional test with proxy ?

**Comment by jordisala1991** [comment]
I dont think so. There are functional tests but not sure if they specifically tests something around the output of this method.

**Pull request merged**